src/main/java/org/jenkinsci/plugins/docker/traceability/model/DockerTraceabilityReportListener.java

```diff
@@ -21,30 +21,34 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.jenkinsci.plugins.docker.traceability;
+package org.jenkinsci.plugins.docker.traceability.model;
```

Comment:
Not sure this should be part of model as it is not part of the JSON odel object.